src/controllers/institution-controller.js

```diff
@@ -0,0 +1,177 @@
+import * as institutionService from '../services/institution-service.js'
+import { models } from '../models/init-models.js'
+import { Sequelize } from 'sequelize'
+import sequelizeConn from '../util/db.js'
+
+export async function fetchProjectInstitutions(req, res) {
+  try {
+    const institutions = await institutionService.fetchInstitutions(
+      req.params.projectId
+    )
+    res.status(200).json({ institutions })
+  } catch (e) {
+    console.error('Problem while getting project institutions (controller).', e)
+    res
+      .status(500)
+      .json({ message: 'Error while getting project institutions.' })
+  }
+}
+export async function addInstitutionToProject(req, res) {
+  const projectId = req.params.projectId
+  const institutionId = req.body.institutionId
```

Comment:
You should **always** check that the user input.

Check to see if the institutionId is non-null and is greater than zero